deps/v8/DEPS

```diff
@@ -143,7 +142,7 @@ deps = {
       }
     ],
     'dep_type': 'cipd',
-    'condition': '(host_os == "linux" or host_os == "mac" or host_os == "win") and checkout_reclient',
+    'condition': 'host_os == "linux" or host_os == "mac" or host_os == "win"',
```

Comment:
@nodejs/platform-ppc @nodejs/platform-s390 This change breaks our V8 CI. What can we do about it (apart floating a revert)?

Reply:
It from this change: https://chromium-review.googlesource.com/c/chromium/src/+/3755181

Reply:
@targos I was pointed to https://chromium-review.googlesource.com/c/v8/v8/+/3992966 by @miladfarca (🙇).

Reply:
@targos The file owner (machenbach) will be back by Nov 3rd. Is this PR able to wait until then or should I try escalating the review process?

Reply:
It can wait.

Reply:
@targos above CL has landed.